interfaces/builtin/ofono.go

```diff
@@ -288,6 +288,15 @@ ATTRS{idVendor}=="1c9e", ATTRS{idProduct}=="9605", ENV{ID_USB_INTERFACE_NUM}=="0
 LABEL="ofono_speedup_end"
 `
 
+const ofonoPermanentSlotUDevTag = `
+KERNEL=="tty[A-RT-Z]*[0-9]|cdc-wdm[0-9]*", TAG+="###SLOT_SECURITY_TAGS###"
+KERNEL=="tun",         TAG+="###SLOT_SECURITY_TAGS###"
+KERNEL=="tun[0-9]*",   TAG+="###SLOT_SECURITY_TAGS###"
+KERNEL=="dsp",         TAG+="###SLOT_SECURITY_TAGS###"
+KERNEL=="ttyLTM[0-9]", TAG+="###SLOT_SECURITY_TAGS###"
+KERNEL=="ttyUSB[0-9]", TAG+="###SLOT_SECURITY_TAGS###"
```

Comment:
I didn't find above three udev rules after digging around.
RE: KERNEL=="modem[0-9]*"

Basically, Linux modem drivers set up the modem device /dev/modem as a symbolic link to the actual device to /dev/ttyS*. So first entry would cover this case.
And /dev/socket/rild is just a socket created by rild daemon when it runs during system boot up. See here
https://stackoverflow.com/questions/11111067/how-does-modem-code-talk-to-android-code/11111953#11111953

I suspect if it's an actual device node we can find in udev database. Same for chnlat.

So I doubt if they're really needed putting here to tag device node which does not actually exist in system.